src/explore-education-statistics-admin/src/pages/users/ManageUserPage.tsx

```diff
@@ -0,0 +1,174 @@
+import Link from '@admin/components/Link';
+import Button from '@common/components/Button';
+import ButtonText from '@common/components/ButtonText';
+import { errorCodeToFieldError } from '@common/components/form/util/serverValidationHandler';
+import {
+  FormFieldset,
+  FormFieldSelect,
+  FormFieldCheckboxGroup,
+  FormFieldTextInput,
+  FormGroup,
+  Formik,
+} from '@common/components/form';
+import Form from '@common/components/form/Form';
+import Page from '@admin/components/Page';
+import useFormSubmit from '@admin/hooks/useFormSubmit';
+import userService from '@admin/services/users/service';
+import { UserStatus } from '@admin/services/users/types';
+import useAsyncRetry from '@common/hooks/useAsyncRetry';
+import Yup from '@common/validation/yup';
+
+import React, { useEffect, useState } from 'react';
+import { Route, RouteComponentProps, Switch } from 'react-router';
+import { IdLabelPair } from 'src/services/common/types';
+
+const errorCodeMappings = [
+  errorCodeToFieldError(
+    'USER_ALREADY_EXISTS',
+    'userEmail',
+    'User already exists',
+  ),
+];
+
+interface Model {
+  user: UserStatus;
+}
+
+interface FormValues {
+  userEmail: string;
+}
+
+const ManageUserPage = ({
+  match,
+  history,
+}: RouteComponentProps<{ userId: string }>) => {
+  const { userId } = match.params;
+  const [model, setModel] = useState<Model>();
+  const formId = userId;
+
+  useEffect(() => {
+    userService.getUser(userId).then(user => {
```

Comment:
Would suggest using the `useAsyncRetry` hook as it handles loading/error states. Ideally we would want to show a `LoadingSpinner` using it's `isLoading` flag.

Reply:
This page is just a mock at the moment so not too concerned at the moent as this page will be fully implemented by BAU-557 and BAU-558.